pkg/beacon/relay/entry/message.go

```diff
@@ -11,6 +11,13 @@ type SignatureShareMessage struct {
 	shareBytes []byte
 }
 
+func NewSignatureShareMessage(
+	senderID group.MemberIndex,
+	shareBytes []byte,
+) *SignatureShareMessage {
+	return &SignatureShareMessage{senderID, shareBytes}
+}
```

Comment:
Instead of adding it here, we should use the trick with `export_test.go`. See `pkg/beacon/relay/gjkr/export_test.go`.

Reply:
We can't use this trick. `integration_tests.go` are in `relay` package and `SignatureShareMessage` in `entry` package.

Reply:
Hmm... right. I was thinking about moving this integration test to `entry` but I see it's painful. Not worth the effort at this point.